### DISCUSSION TITLE
QQ频道如何获取没有艾特机器人的消息

### DISCUSSION THREAD

**Discussion by HFYGitHub** [simple-robot/Discussions#185] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

QQ频道

### 疑问描述

我在QQ频道发消息，用ChannelMessageEvent监听，机器人只能收到艾特的消息，有什么办法能获取频道的所有消息吗

**Comment by ForteScarlet** [comment]
如果你的机器人是 **公域** 机器人，那么只能收到at消息，全量消息是私域的

参考：[事件订阅 intents](https://bot.q.qq.com/wiki/develop/api/gateway/intents.html)
